src/project/ProjectModel.js

```diff
@@ -991,6 +991,7 @@ define(function (require, exports, module) {
             viewModel.renameItem(oldProjectPath, self.makeProjectRelativeIfPossible(newPath));
             if (self._selections.selected && self._selections.selected.indexOf(oldPath) === 0) {
                 self._selections.selected = newPath + self._selections.selected.slice(oldPath.length);
+                self.setCurrentFile(newPath);
```

Comment:
Whats the reason behind _selections.selected being null in second drag?

Reply:
@navch This [line](https://github.com/adobe/brackets/blob/master/src/project/ProjectModel.js#L627) sets selection to null

Reply:
👍 Sounds reasonable.